x-pack/plugins/maps/public/classes/layers/blended_vector_layer/blended_vector_layer.ts

```diff
@@ -177,9 +179,9 @@ export class BlendedVectorLayer extends VectorLayer implements IVectorLayer {
   }
 
   private readonly _isClustered: boolean;
-  private readonly _clusterSource: IESAggSource;
+  private readonly _clusterSource: ESGeoGridSource;
```

Comment:
Since `_clusterSource` is now `ESGeoGridSource`, should the signature of `getClusterStyleDescriptor` be updated to accept `ESGeoGridSource` instead of `IESAggSource`?